grpc/test_proto/interceptors_test.proto

```diff
@@ -2,11 +2,16 @@ syntax = "proto3";
 
 option go_package = "github.com/letsencrypt/boulder/grpc/test_proto";
 
+import "google/protobuf/timestamp.proto";
+
 service Chiller {
   // Sleep for the given amount of time, and return the amount of time slept.
   rpc Chill(Time) returns (Time) {}
 }
 
 message Time {
+  // Next unused field number: 3
   int64 timeNS = 1; // Unix timestamp (nanoseconds)
+  // TODO (#7097) Change this to google.protobuf.Duration
+  google.protobuf.Timestamp time = 2;
```

Comment:
Why is this change in this PR? I don't see anything using this new field, so we might as well leave it out until it's time to do #7097.

Reply:
You're right, this proto file exists just to create a test GRPC client that [sleeps for a specific time](https://github.com/letsencrypt/boulder/blob/3dbeaf7da072bcf0e7bfcba15acdd1084522ef6f/grpc/interceptors_test.go#L147).